testsuite/gateways/wasm/__init__.py

```diff
@@ -76,6 +77,8 @@ def add_mapping_rules(self, service, rules):
     def _create_api_client(self, application, endpoint, verify, cert=None, disable_retry_status_list=None):
         ext = self.extensions[application.service["id"]]
         ext.synchronise_mapping_rules()
+        ext.synchronise_credentials()
+        sleep(10)  # wait needed for WASM to load new configuration
```

Comment:
Would something like this `ext.httpbin.deployment(f"dc/{ext.httpbin_name}").wait_for()` work too? Even though I hate the idea of redeploying httpbin due to config change, I hate the idea of `sleep(10)` more

Reply:
Tried this, but sadly was not enough.